.cirrus.yml

```diff
@@ -688,7 +691,8 @@ upgrade_test_task:
     name: "Upgrade test: from $PODMAN_UPGRADE_FROM"
     alias: upgrade_test
     skip: *tags
-    only_if: $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*' || $CIRRUS_CRON != ''
+    # Only run during daily cirrus-cron
```

Comment:
Wait, is this comment accurate? I'm confused.

Reply:
woopsie